model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -248,7 +248,10 @@ object SortDirections {
 
   def toSql(direction: SortDirection) = toString(direction)
 }
-case class EntityQuery(page: Int, pageSize: Int, sortField: String, sortDirection: SortDirections.SortDirection, filterTerms: Option[String])
+case class EntityQuery(page: Int, pageSize: Int,
+                       sortField: String, sortDirection: SortDirections.SortDirection,
+                       filterTerms: Option[String],
+                       fields: WorkspaceFieldSpecs = WorkspaceFieldSpecs())
```

Comment:
is it worth renaming `WorkspaceFieldSpecs` to something more generic? Or does that really matter?

Reply:
I think that change will also impact the PR I currently have open. Maybe we change it once both are merged in a separate small PR?

Reply:
works for me! I'd prefer to do it in a standalone PR anyway to avoid the noise